pontoon/urls.py

```diff
@@ -40,6 +41,9 @@
     url(r'^pontoon\.js$',
         RedirectView.as_view(url='/static/js/pontoon.js', permanent=True)),
 
+    # Robots.txt
+    (r'^robots.txt$', lambda r: HttpResponse("User-agent: *\nDisallow: /a/\nDisallow: /admin/", content_type="text/plain")),
```

Comment:
This totally works, but I'd rather have this in a template file for maintainability's sake. A simple view that just calls render should be all that's needed.